host-configs/quartz-toss_4_x86_64_ib-clang@14.0.6.cmake

```diff
@@ -0,0 +1,108 @@
+  set(CMAKE_Fortran_COMPILER "/usr/WS1/axom/libs/toss_4_x86_64_ib/2023_05_18_07_57_03/spack/lib/spack/env/clang/gfortran" CACHE PATH "")
+
+else()
+
+  set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-14.0.6/bin/clang" CACHE PATH "")
+
+  set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-14.0.6/bin/clang++" CACHE PATH "")
+
+  set(CMAKE_Fortran_COMPILER "/usr/tce/packages/gcc/gcc-10.3.1/bin/gfortran" CACHE PATH "")
+
+endif()
+
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+
+set(BLT_EXE_LINKER_FLAGS " -Wl,-rpath,/usr/tce/packages/clang/clang-14.0.6/lib" CACHE STRING "Adds a missing libstdc++ rpath")
+
+#------------------------------------------------------------------------------
+# MPI
+#------------------------------------------------------------------------------
+
+set(MPI_C_COMPILER "/usr/tce/packages/mvapich2/mvapich2-2.3.6-clang-14.0.6/bin/mpicc" CACHE PATH "")
+
+set(MPI_CXX_COMPILER "/usr/tce/packages/mvapich2/mvapich2-2.3.6-clang-14.0.6/bin/mpicxx" CACHE PATH "")
+
+set(MPI_Fortran_COMPILER "/usr/tce/packages/mvapich2/mvapich2-2.3.6-clang-14.0.6/bin/mpif90" CACHE PATH "")
+
+set(MPIEXEC_EXECUTABLE "/usr/bin/srun" CACHE PATH "")
+
+set(MPIEXEC_NUMPROC_FLAG "-n" CACHE STRING "")
+
+set(ENABLE_MPI ON CACHE BOOL "")
+
+#------------------------------------------------------------------------------
+# Hardware
+#------------------------------------------------------------------------------
+
+#------------------------------------------------
+# Hardware Specifics
+#------------------------------------------------
+
+set(ENABLE_OPENMP ON CACHE BOOL "")
+
+set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "")
+
+#------------------------------------------------------------------------------
+# TPLs
+#------------------------------------------------------------------------------
+
+set(TPL_ROOT "/usr/WS1/axom/libs/toss_4_x86_64_ib/2023_05_18_07_57_03/clang-14.0.6" CACHE PATH "")
+
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.6-qnuy4j5xgko2zk3c3jdmpww2vhhoka4d" CACHE PATH "")
+
+set(C2C_DIR "${TPL_ROOT}/c2c-1.3.0-suwgwulvsi2a4ghsejbkzhw6nvcavlwm" CACHE PATH "")
+
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.5.0-npldlsboltqnbu4yxlr7esb3bfpi6x56" CACHE PATH "")
+
+# HDF5 not built
+
+set(LUA_DIR "/usr" CACHE PATH "")
+
+set(RAJA_DIR "${TPL_ROOT}/raja-2022.03.0-jjcsjmscrhxcksd7m26q462vhhgicylf" CACHE PATH "")
+
+set(UMPIRE_DIR "${TPL_ROOT}/umpire-2022.03.1-eri46kk5td3v23xgppxxwxovlusgm4zn" CACHE PATH "")
+
+set(CAMP_DIR "${TPL_ROOT}/camp-2022.10.1-ug3vynwpuw3zbkvpscygelocijatm3fu" CACHE PATH "")
+
+# scr not built
+
+#------------------------------------------------------------------------------
+# Devtools
+#------------------------------------------------------------------------------
+
+set(DEVTOOLS_ROOT "/collab/usr/gapps/axom/devtools/toss_4_x86_64_ib/2023_05_18_00_10_54/._view/if3mvrcfyezs6ftljp5agtvcgzq24dsy" CACHE PATH "")
+
+set(CLANGFORMAT_EXECUTABLE "${DEVTOOLS_ROOT}/llvm-10.0.0/bin/clang-format" CACHE PATH "")
+
+set(ENABLE_CLANGFORMAT OFF CACHE BOOL "")
```

Comment:
> set(ENABLE_CLANGFORMAT OFF CACHE BOOL "")

Is there a reason we can't get clang-format on toss4? 
Or is this just a temporary restriction while you work through issues w/ our devtools?

Reply:
because i messed up and the fix is now coming.

Reply:
Thanks!